src/redux/api/slice.js

```diff
@@ -18,6 +21,51 @@ export const apiSlice = createApi({
         body: {},
       }),
     }),
+    getUnassignedTasks: builder.query({
+      async queryFn(date, _queryApi, _extraOptions, fetchWithBQ) {
+        const result = await fetchAllRecordsUsingFetchWithBQ(
+          fetchWithBQ,
+          'api/tasks',
+          100,
+          {
+            date: date.format('YYYY-MM-DD'),
+            assigned: 'no'
+          });
+
+        return result ? { data: result } : { error: "result.error" };
+      }
+    }),
+    getTaskLists: builder.query({
+      async queryFn(date, _queryApi, _extraOptions, fetchWithBQ) {
+        const result = await fetchAllRecordsUsingFetchWithBQ(
+          fetchWithBQ,
+          'api/task_lists',
+          100,
+          {
+            date: date.format('YYYY-MM-DD'),
+          });
+
+        return result ? { data: result } : { error: "result.error" };
+      },
+    }),
+    getCourierUsers: builder.query({
+      async queryFn(_arg, _queryApi, _extraOptions, fetchWithBQ) {
+        const result = await fetchAllRecordsUsingFetchWithBQ(
+          fetchWithBQ,
+          '/api/users',
```

Comment:
For this endpoint, pagination has to be enabled explicitly (after this PR is merged: https://github.com/coopcycle/coopcycle-web/pull/4929 also it will enable this endpoint for role Dispatcher)